lib/receiver/receiver_impl.cc

```diff
@@ -192,7 +192,14 @@ namespace gr
         sch_search_handler(input, noutput_items);
         break;
       case synchronized:
-        synchronized_handler(input, input_items, noutput_items);
+	synchronized_handler(input, input_items, noutput_items,0);
+	size_t inputs_to_process = d_cell_allocation.size();
+	if (d_process_uplink)
+        	inputs_to_process *= 2;
+	for(int input_nr=0;input_nr<inputs_to_process;input_nr++){
```

Comment:
I think the loop should start from 1 because the 0 channel is processed few lines above.